base/uk.ac.stfc.isis.ibex.opis/resources/opi_info.xml

```diff
@@ -959,5 +959,47 @@
 				</macros>
 			</value>
 		</entry>
+		<entry>
+			<key>Sample stack</key>
```

Comment:
I know this was probably done before the categorization stuff was done, but would you mind merging in master and putting this OPI into the relevant category so that it displays nicely in the synoptic editor?

Reply:
No problem